src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -0,0 +1,45 @@
+package frc.team2412.robot;
+
+import frc.team2412.robot.subsystems.*;
+
+import static frc.team2412.robot.Subsystems.SubsystemConstants.*;
+
+public class Subsystems {
+    public static class SubsystemConstants {
+        public static final boolean CLIMB_ENABLED = false;
+        public static final boolean DRIVE_ENABLED = false;
+        public static final boolean FRONT_VIS_ENABLED = false;
+        public static final boolean GOAL_VIS_ENABLED = false;
+        public static final boolean INDEX_ENABLED = false;
+        public static final boolean INTAKE_ENABLED = false;
+        public static final boolean SHOOTER_ENABLED = false;
```

Comment:
These should be private (or private protected), as only the enclosing Subsystems class needs to use them.